showpass/showpass-wordpress-plugin
# fix(wordpress) | SPW 7280 - Fix image resizer, embedded calendar and external link API URLs

This fixes a bug where the production image resizer domain, production API for the embedded calendar and production API for external links with distribution config is being used despite the plugin being configured to use the beta API

**How to test**

- configure the plugin to connect to beta.showpass.com
- Put in an organization ID from beta
- Go to a page and test the different shortcodes and templates: 
- https://github.com/showpass/showpass-wordpress-plugin/blob/master/README.md


Note: Tests are failing in circlCI atm due to issues with the composer.lock file.  This should be addressed in a different PR instead.
